File: PPA/DailyFlash/July/24July/Program2.c


#include<stdio.h>

void main() {
	
	for(int i=1; i<=5; i++) {
		for(int j= 4; j>=i; j--) {

			printf("  ");
		}

		int num=0;
		for(int k=1; k<=i;k++) {

			if(num==0) {
				printf("%d ",num);
				num++;
			}
			else {
				printf("%d ",num);
				num--;
			}

		}
		printf("\n");
	}
}
			

File: Java/Assignments/Assignment/IOAssignment2-1/Program2.java

import java.io.*;

class Divisor {

	public static void main(String[] args) throws IOException {

		InputStreamReader isr = new InputStreamReader(System.in);
		BufferedReader br = new BufferedReader(isr);

		int num;

		System.out.print("Enter a number: ");
		num=Integer.parseInt(br.readLine());

		for(int i=1; i<=num; i++) {

			if(num%i==0)
				System.out.print(i+" ");
		}
		System.out.println();
	}
}
